src/components/tasks/card/index.tsx

```diff
@@ -325,8 +329,8 @@ const Card: FC<Props> = ({
             className={classNames.contributorImage}
           >
             <Image
-              src={assigneeProfilePic}
-              alt="Assignee profile picture"
+              src={assigneeProfilePic || '/dummyProfile.png'}
```

Comment:
Would it be better to keep the default value of `assigneeProfilePic` state as `/dummyProfile.png` instead of short-circuting it here? Same for `alt` text as well.

Reply:
+1

Reply:
ok i will remove it.